src/shared/is_node.js

```diff
@@ -14,15 +14,18 @@
  */
 /* globals process */
 
-// NW.js / Electron is a browser context, but copies some Node.js objects; see
+// NW.js / Electron / BrightSign is a browser context,
+// but copies some Node.js objects; see
 // http://docs.nwjs.io/en/latest/For%20Users/Advanced/JavaScript%20Contexts%20in%20NW.js/#access-nodejs-and-nwjs-api-in-browser-context
 // https://www.electronjs.org/docs/api/process#processversionselectron-readonly
 // https://www.electronjs.org/docs/api/process#processtype-readonly
+// https://brightsign.atlassian.net/wiki/spaces/DOC/pages/370672370/BSDeviceInfo
 const isNodeJS =
   (typeof PDFJSDev === "undefined" || PDFJSDev.test("GENERIC")) &&
   typeof process === "object" &&
   process + "" === "[object process]" &&
   !process.versions.nw &&
-  !(process.versions.electron && process.type && process.type !== "browser");
+  !(process.versions.electron && process.type && process.type !== "browser") &&
+  !(window && window.BSDeviceInfo);
```

Comment:
Given that `window` isn't defined in *proper* Node.js environments, won't this line completely break things in that case?
At the very least, you'd have to check `typeof window !== "undefined" to be safe.

Reply:
I fix it. Thank you.